### DISCUSSION TITLE
Set rotation center

### DISCUSSION THREAD

**Discussion by madsenfr** [SVG-Edit/svgedit#804] [New feature request]
## Motivation

It would be easyer to produce the correct shape, by choosing rotation center and then setting postion.

## Current behavior

Actually rotation center is center of the figure.

## Desired behavior

To be able to set x and y center of rotation.

## Alternatives considered

When editing code and changing rotation center, it works, until I click on the shape.
Then center return to center of the shape.

**Comment by distler** [comment]
UI aside, what does that correspond to in terms of the [`transform` attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform) of the element in question?

**Comment by madsenfr** [comment]
It's the second and third parameters of the rotate attribute.

`<rect x="200" y="200" width="100" height="100" transform="rotate(45 250 250)" fill="#000000"/>`

**250 250** cannot be changed for now.
